### PR TITLE
Feat/user stories

### PR DESCRIPTION
**Pull request by jvargas98**
1.- As a user, i want to be able to upload a profile picture, so that my user feels more unique and personal
  Acceptance points:

- The picture must be in .jpeg, .png or .jpg format
- The picture max size is 900 x 900 px

2.-  As an user, i want to be able to save times or "laps" with a name in the stopwatch, so that i can have a record of my times.
Acceptance points:
 - A Stopwatch can have zero, or N number of laps
 - If the user doesn't write a name for the laps saved, the default name must be the current date

3.- As an user, i want to be able to set timers, so that i can have another tool that helps me with my day to day activities
  Acceptance points:
 - The format of the timer has to be
        00     :    00      :      00
      Hours | Minutes | Seconds
 - When the timer finishes, it must alert the user
 - The user can pause the timer and stop it completly

### CODE REVIEW

**Review by GregVaz — APPROVED** [review]
Great job

File: app/javascript/packs/clock.js [hunk 1]
```diff
@@ -53,28 +73,45 @@ function reset() {
...
 // Create function to display buttons
 
 function showButton(buttonKey) {
   if (buttonKey === "PLAY") {
-    playButton.classList.remove("d-none")
-    pauseButton.classList.remove("d-block")
-    pauseButton.classList.add("d-none")
+    playButton.classList.remove("d-none");
+    pauseButton.classList.remove("d-block");
+    pauseButton.classList.add("d-none");
```
Comment (ese-varo) [inline app/javascript/packs/clock.js:98]: I think you could achieve the same result with just one line, but you would need to add all the classes that you need for that tag: `pauserButton.className('class-1 class-2')`

The same could apply for similar scenarios you have down below when you add and remove classes on the same tag

**Review by LoginALee — COMMENTED** [review]
Great work Jorge! :100:

File: app/javascript/packs/timer.js [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    remainSeconds,
+    remainMinutes,
+    remainHours,
+    remainDays,
+    remainTime
+  }
+};
+
+const countdown = (deadline, elem, finalMessage, id) => {
+  console.log(id)
```
Comment (LoginALee) [inline app/javascript/packs/timer.js]: Do you still need that console.log ? or it was just for testing?

File: app/javascript/packs/timer_1.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+  return `${formattedHH}:${formattedMM}:${formattedSS}`;
+}
+
+
+function setAlarm(timerId) {
+  let timer = getTimerTime(timerId);
+  let diff = timer.getTime() - (new Date("01/01/1970 ")).getTime();
+
+  console.log(diff);
```
Comment (LoginALee) [inline app/javascript/packs/timer_1.js]: Same here

File: app/javascript/packs/clock.js (hunk 1 above)
Comment (ese-varo) [reply app/javascript/packs/clock.js:94]: Besides that, you did a good job! :rocket: :100:

**Comment by ese-varo** [comment]
I think some parts of your code could be cleaner specifically on js, it seems that there is some logic repeated around your functions (like appending elements to the same tag), but Besides that, you did a really great job! :rocket: :100:

File: app/javascript/packs/clock.js (hunk 1 above)
Comment (jvargas98) [reply app/javascript/packs/clock.js:98]: Thanks

File: app/javascript/packs/timer.js (hunk 2 above)
Comment (jvargas98) [reply app/javascript/packs/timer.js]: Thanks for the observation

**Pull request merged**